### PR TITLE
161780573 lists all users

### PR DESCRIPTION
**Pull request by kayroy247**
**Description**
- Lists all users 

**Type of Change**
- Feature

**How has this been tested**
- This has been tested with test driven development
- It has also been tested with Postman

**Pivotal Tracker ID
161780573

### CODE REVIEW

**Review by Fiyiin — COMMENTED** [review]
LGTM

File: src/routes/api/users/UserController/UserController.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+   * @param {obj} res
+   * @returns {obj} res
+   * @memberof Users
+   */
+  static async getAllUsers(req, res) {
+    const result = await db.user
+      .findAll({
+        attributes: ['username', 'firstname', 'lastname', 'email', 'updatedAt', 'createdAt', 'bio', 'image']
+      });
+    delete result.password;
```
Comment (profmcdan) [inline src/routes/api/users/UserController/UserController.js]: Attention to details? Why is line 22 still here. Mutation?

File: src/routes/api/users/UserController/UserController.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+   *
+   *
+   * @static
+   * @param {obj} req
+   * @param {obj} res
+   * @returns {obj} res
+   * @memberof Users
+   */
+  static async getAllUsers(req, res) {
+    const result = await db.user
```
Comment (profmcdan) [inline src/routes/api/users/UserController/UserController.js]: result? You could have simply used users

**Review by profmcdan — CHANGES_REQUESTED** [review]
Please review changes

File: src/routes/api/users/UserController/UserController.js (hunk 2 above)
Comment (kayroy247) [reply src/routes/api/users/UserController/UserController.js]: Thanks for the feedback. I have implemented it.

File: src/routes/api/users/UserController/UserController.js (hunk 1 above)
Comment (kayroy247) [reply src/routes/api/users/UserController/UserController.js]: Thanks for the feedback. I have removed that.

File: src/routes/api/users/UserController/UserController.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+   * @static
+   * @param {obj} req
+   * @param {obj} res
+   * @returns {obj} res
+   * @memberof Users
+   */
+  static async getAllUsers(req, res) {
+    const users = await db.user
+      .findAll({
+        attributes: ['username', 'firstname', 'lastname', 'email', 'updatedAt', 'createdAt', 'bio', 'image']
```
Comment (profmcdan) [inline src/routes/api/users/UserController/UserController.js]: Change attributes to exclude

**Review by profmcdan — CHANGES_REQUESTED** [review]
Review

File: src/routes/api/users/UserController/UserController.js (hunk 3 above)
Comment (kayroy247) [reply src/routes/api/users/UserController/UserController.js]: Thanks for the feedback. I have implemented the feedback.

**Review by profmcdan — APPROVED** [review]

**Pull request merged**